files/en-us/learn/css/building_blocks/debugging_css/index.html

```diff
@@ -145,7 +145,7 @@ <h3 id="Do_you_have_valid_HTML_and_CSS">Do you have valid HTML and CSS?</h3>
  <li><a href="https://validator.w3.org/">HTML validator</a></li>
 </ul>
 
-<h3 id="Is_the_property_and_value_supported_by_the_browser_you_are_testing_in">Is the property and value supported by the browser you are testing in?</h3>
+<h3 id="Is_the_property_and_value_supported_by_the_browser_you_are_testing_in">Are the property and value supported by the browser you are testing in?</h3>
```

Comment:
```suggestion
<h3 id="Are_the_property_and_value_supported_by_the_browser_you_are_testing_in">Are the property and value supported by the browser you are testing in?</h3>
```

> Looks like the ID of the h3 will need to be updated to, but hopefully that will be automatic...

Unfortunately not :).